### PR TITLE
HttpStress: add a 'GET Slow' operation

### PR DESCRIPTION
**Pull request by eiriktsarpalis**
* Adds a 'GET Slow' operation.
* Randomize the response content length in GET requests. Results are validated by having the client specify the expected length in request headers.

### CODE REVIEW

File: src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs [hunk 1]
```diff
@@ -158,37 +166,28 @@ public static class ClientOperations
...
-                        using (Stream s = await m.Content.ReadAsStreamAsync())
-                        {
-                            s.ReadByte(); // read single byte from response and throw the rest away
-                        }
+                    ValidateHttpVersion(m, ctx.HttpVersion);
+                    ValidateStatusCode(m);
+
+                    using (Stream s = await m.Content.ReadAsStreamAsync())
+                    {
+                        s.ReadByte(); // read single byte from response and throw the rest away
```
Comment (scalablecory) [inline src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs]: Is this intended to be sync?

File: src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```
[the remark this answers is not in the document]
Comment (eiriktsarpalis) [reply src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs]: Fixed

File: src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs (hunk 1 above)
Comment (eiriktsarpalis) [reply src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs]: Probably not. I'm assuming @stephentoub wrote it like this for brevity.

Comment (stephentoub) [reply src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs]: Right, I didn't really care in this case whether it was sync or async, I just wanted simple. Feel free to change it if you please, or leave it, either way :)

File: src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs [hunk 3]
```diff
@@ -158,67 +166,54 @@ public static class ClientOperations
...
-                                }
-                            }
-                        }
+                    ValidateHttpVersion(res, ctx.HttpVersion);
+                    ValidateStatusCode(res);
 
+                    // Validate that request headers are being echoed
+                    foreach (KeyValuePair<string, IEnumerable<string>> reqHeader in req.Headers)
+                    {
+                        if (!res.Headers.TryGetValues(reqHeader.Key, out var values))
```
Comment (scalablecory) [inline src/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs]: Type of `var` isn't obvious here.

```suggestion
                        if (!res.Headers.TryGetValues(reqHeader.Key, out IEnumerable<string> values))
```

**Pull request merged**